### PR TITLE
fix panel, fix web api routes, fix raw message in broadcast service, add tests

### PR DESCRIPTION
**Pull request by amlcodes**

### CODE REVIEW

File: src/services/broadcast-service.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ class BroadcastService {
       );
       logger.debug(`send1to1Messge returns=${uMessage}`);
       reply.pending = 'Broadcast message in process of being sent to list of users';
-      reply.message = messageToSend;
+      reply.message = this.user.message;
```
Comment (pwcushman) [inline src/services/broadcast-service.js:132]: This is not a valid change, it should be messageToSend.  This value was generated based on the message, and the acknowledgement setting, etc...

Comment (amlcodes) [reply src/services/broadcast-service.js:132]: is something else readying this full message value (message, ack, and sentby) being returned by the broadcast service.sendMessage? My frontend needs the raw message?

Comment (amlcodes) [reply src/services/broadcast-service.js:132]: I scanned the codebase, it doesn't look like anything else reads the .message value from broadcast message so it should be safe to touch

Comment (amlcodes) [reply src/services/broadcast-service.js]: I added rawMessage, and changed message back to messageToSend

**Pull request merged**